examples/SharedMemory/PhysicsServerCommandProcessor.cpp

```diff
@@ -5022,8 +5022,10 @@ bool PhysicsServerCommandProcessor::processSyncUserDataCommand(const struct Shar
 
 	b3AlignedObjectArray<int> userDataHandles;
 	m_data->m_userDataHandles.getUsedHandles(userDataHandles);
-	memcpy(bufferServerToClient, &userDataHandles[0], sizeof(int) * userDataHandles.size());
-
+	if (userDataHandles.size())
```

Comment:
Is the check really necessary? I was under the impression that if the size is zero, `memcpy` would copy nothing...

Reply:
The check is to avoid accessing non-existing elements in the array (userDataHandles[0], while the size == 0). It causes asserts in debug mode.